Apps/WebClient/src/ClientApp/src/views/ProfileView.vue

```diff
@@ -133,6 +132,11 @@ export default class ProfileView extends Vue {
 
     private intervalHandler = 0;
 
+    private debounceTimeouts: Map<string, NodeJS.Timeout> = new Map<
```

Comment:
will that work in a browser?

Reply:
yes, I tested, but I did find it weird at first, here is a better solution:

https://stackoverflow.com/questions/45802988/typescript-use-correct-version-of-settimeout-node-vs-window

I'll update accordingly

Reply:
this is no longer an issue removed debounce and used a combination of regex and endpoint calling to throttle junk calls.